### PR TITLE
added sandbox level

### PR DESCRIPTION
**Pull request by Jannikergudingenreklamus**
- [X] *I confirm that I have read and applied my submission according to the [Contributing Guidelines](https://github.com/CorsixTH/CommunityContent/blob/main/CONTRIBUTING.md)*

# Community Content Contribution (choose one)
- [X] I am submitting a standalone level
- [ ] I am submitting a map only
- [ ] I am submitting a campaign

**Name of level/map/campaign : Sandbox**
**(If relevant) Levels of campaign:1 **

## Campaign Data
- [x] Not applicable
- [ ] I have included a `.campaign` file with this submission

## Game Data
**Please note: All level files used MUST point to a map file to be accepted**

**Level Name:**
The `.level` file is:
- [ ] Not applicable, map only
- [X] From CorsixTH (specify level file name)
example.map
- [ ] From a Community Content submission (specify level file name and original author)
- [X] My own creation and is included in this submission

*Level Declarations (as guided above):*

The `.map` file is:
- [ ] From CorsixTH (specify level file name)
- [ ] From a Community Content submission (specify level file and original author)
- [X] My own creation and is included in this submission

*Map Declarations (as guided above):*

### Your content/files
**Note:** Your authorship declarations must also contain the required reference to the Creative Commons Attribution 4.0 International License (CC BY 4.0)
- [X] *I confirm that I am the author of this work.*
- [X] *I confirm to have put appropriate authorship on all submitted files (including
modified works).*
- [X] *I confirm to have included a `LICENSE.txt` file with my submission.*
### Derivative/redistributed content (complete if applicable)

- [X] *Where portions of this work have been created by other authors (include works
I have modified) I have appropriately declared their authorship and license in the
relevant files.*
- [X] *I confirm that any works from other authors allow for re-use and modification.*
### Agreement
- [X] *I agree for the works to be licensed under Creative Commons Attribution 4.0 International - https://creativecommons.org/licenses/by/4.0/*
- [X] *I confirm that my work is free from offensive (or otherwise sensitive) content.*
- [X] *I understand that CorsixTH will not be responsible for enforcement of the CC BY 4.0 license
except for submissions of falsified authorship (in which case, the offending works
will be removed).*
- [X] *I confirm that I am happy for this work to be uploaded and distributed by the
CorsixTH Community Content repository.*

### CODE REVIEW

**Comment by TheCycoONE** [comment]
Bear with us as this is the first community contribution, but I believe there are a few things to address:

The Map directory entry is only for maps with no attached level file. You don't need to include your map twice.

You must provide a license for your map and level per the contributing guidelines.

I believe we expect the level to be in it's own subdirectory, this would be necessary if it's to contain it's own LICENSE.txt

**Comment by Jannikergudingenreklamus** [comment]
sorry.
licence: i noticed was already a file named license in folders. so i
assumed that just not editing would be ok.
i dont care for any licencing or naming. i made this map only to help other
people who wanted a sandbox level
i saw license.txt in all folders and thought that must be enough.
and your guide asked me to put files in folders as described in guide. (is
first time i ever try fork)

i playtested map fully filled hospital with rooms and over 80 patients no
errors popping up. compared to last version wich had pathfinding errors
popups popping up withing 20 minutes.

this map can be considered playtested and open for sandbox.
anyone are wellcome to reupload as your own.

Den søn. 8. jan. 2023 kl. 16.48 skrev Stephen E. Baker <
***@***.***>:

**Comment by TheCycoONE** [comment]
Sure, don't touch anything yet - we'll discuss it. Frankly the license stuff is as much about protecting us as you.

File: Levels/sandboxj.level [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+By default there are no diseases and only some basic rooms are available.
+
+
+---------------------- General Information -------------------------
+%Name = "Sandbox"
+%MapFile = "sandboxj.map"
+
+Everything within citation marks will be the briefing/debriefing.
+Two new lines creates a new paragraph, but additional new lines in a row are discarded.
+%LevelBriefing = "Sandboxmap build build build"
```
Comment (lewri) [inline Levels/sandboxj.level]: Just to verify, is this the only section where text has been changed? Or are these other parts of this file also changed?

File: Levels/sandboxj.level [hunk 2]
```diff
@@ -0,0 +1,248 @@
+Copyright (c) 2010 Edvin "Lego3" Linge
```
Comment (lewri) [inline Levels/sandboxj.level]:
```suggestion
Copyright (c) 2010 Edvin "Lego3" Linge, 2023 Jannikergudingenreklamus
```

**Review by lewri — COMMENTED** [review]
`Maps/sandboxj.map` can be removed from the PR. *However*, I do get why it would have been useful there for other people as a generic megamap.

`sandboxj.level` and `sandboxj.map` to be in their own folder `Sandboxj` inside `Levels`

A license.txt file needs adding to the aforementioned folder `Sandboxj`


**No action required here you on part right now, this is just my understanding on what is missing/ needs adjusting**

**Pull request merged**